main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -369,7 +369,6 @@
     "core-project/starting": "Démarrer",
     "core-project/paste-json": "Coller une liste d’opérations au format JSON à appliquer :",
     "core-project/undo-history": "Historique d’annulation infinie",
-    "core-project/discard": "Supprimer",
```

Comment:
Maybe just rename? `core-project/hide-panel`?

Reply:
Ok but what about the value? I'm not sure my translation is right.

Reply:
> Ok but what about the value? I'm not sure my translation is right.

I'm a native francophone, I can assure you. ;-) And Antonin is French! You are in good hands with us. ;-)

Regards, Antoine

Reply:
Awesome!